src/build/dotnet/tasks/netcore/assembly-sign.yaml

```diff
@@ -16,7 +17,7 @@ steps:
       targetType: inline
       pwsh: true
       script: |
-        $project = "${{ parameters.projects }}"; # todo: what about multi-line values here
+        $project = "${{parameters.projects}}"; # todo: what about multi-line values here
```

Comment:
this TODO is disconcerting - any idea what it means?

Reply:
I have no idea sorry, was unable to work out what it meant when I saw this.